src/v3/src/components/Form/renderers.tsx

```diff
@@ -167,21 +167,22 @@ export default [
   {
     tester: ({
       options: {
-        inputMeta: { options } = {} as Input,
+        inputMeta: { options, type } = {} as Input,
         format,
         customOptions,
       },
-    }: FieldElement) => (Array.isArray(customOptions) || Array.isArray(options)) && format === 'radio',
+    }: FieldElement) => (Array.isArray(customOptions) || Array.isArray(options)) && ([format, type].includes('radio')),
```

Comment:
Customers can send in `radio` as the `Input` type and it is expected to be respected and render as such (tested against v2).